README.md

```diff
@@ -4,128 +4,103 @@
 
-## Office Hours
+## Dependencies 
 
-Members of the M3 team will hold office hours on the third Thursday of every month from 11-1pm EST. In order to join, make sure to sign up for a slot here: [https://calendly.com/chronosphere-intro/m3-community-office-hours](https://calendly.com/chronosphere-intro/m3-community-office-hours).
+The simplest and quickest way to try M3 is to use Docker, read the M3 getting started section for other options. 
 
-## Test it out
+```
+docker pull quay.io/m3db/m3dbnode:latest
+```
 
-The easiest way to testing out M3 is to follow one of the guides from the documentation. For a fully comprehensive getting started guide, see our [single node how-to](https://docs.m3db.io/how_to/single_node/).
+## Usage 
 
-### Starting a node
+The below is a simplified version of the [M3 quickstart guide](https://m3db.io/docs/quickstart/docker/), and we suggest you read that for more details.
 
+1. Start a container
 ```
-# to build a local m3dbnode process
-make m3dbnode (note that we currently require at least Go 1.10 or higher)
-
-# run it with the sample configuration
-./bin/m3dbnode -f ./src/dbnode/config/m3dbnode-local-etcd.yml
+docker run -p 7201:7201 -p 7203:7203 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3db/m3dbnode:v1.0.0
 ```
 
-To cross-compile and build for Linux AMD64 build with `make m3dbnode-linux-amd64`.
-
-### Creating a namespace to store metrics
-
+2. Create a placement and namespace
 ```
+#!/bin/bash
 curl -X POST http://localhost:7201/api/v1/database/create -d '{
   "type": "local",
   "namespaceName": "default",
-  "retentionTime": "2h"
-}'
+  "retentionTime": "12h"
+}' | jq .
 ```
 
-### Test RPC
-
-To test out some of the functionality of M3DB there are some user friendly HTTP JSON APIs that you can use. These use the DB node cluster service endpoints.
-
-Note: performance sensitive users are expected to use the more performant endpoints via either the Go `src/dbnode/client/Session` API, or the GRPC endpoints exposed via `src/coordinator`.
-
-#### Write a datapoint
+3. Ready a namespace
+```
+curl -X POST http://localhost:7201/api/v1/services/m3db/namespace/ready -d '{
+  "name": "default"
+}' | jq .
+```
 
+4. Write metrics
 ```
-curl http://localhost:9003/writetagged -s -X POST -d '{
-  "namespace": "default",
-  "id": "foo",
-  "tags": [
-    {
-      "name": "__name__",
-      "value": "user_login"
-    },
+#!/bin/bash
+curl -X POST http://localhost:7201/api/v1/json/write -d '{
+  "tags": 
     {
-      "name": "city",
-      "value": "new_york"
+      "__name__": "third_avenue",
+      "city": "new_york",
+      "checkout": "1"
     },
-    {
-      "name": "endpoint",
-      "value": "/request"
-    }
-  ],
-  "datapoint": {
-    "timestamp":'"$(date +"%s")"',
-    "value": 42.123456789
-  }
+    "timestamp": '\"$(date "+%s")\"',
+    "value": 3347.26
 }'
 ```
 
-#### Query for reverse indexed time series data
-
+5. Query results
```

Comment:
There's a slight issue with these commands thanks to the different ways that OSs handle dates. So actually we need to add two examples here annoyingly.